src/emmy/abstract/function.cljc

```diff
@@ -139,6 +140,13 @@
    (defmethod print-method Function [^Function f ^java.io.Writer w]
      (.write w (.toString f))))
 
+#?(:clj
```

Comment:
where is this used? Did we have output were functions were ugly in the ordinary case? (curiosity only: it can be hard in Clojure-land to find out who has intervened in a situation like this)

Reply:
nvm I see in the test below. Is it fair to say that this is a workaround for a bug in `pprint`? I would think that once we had implemented `toString` it would be honored equally by [p]print

Reply:
yes! ugh, I should have written this down... i was getting a `LazySeq` output... but `pprint` was deferring to some other method than our `toStr` here. I should justify this...